modules/installation-bootstrap-gather.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-bootstrap-gather-{context}"]
+= Gathering logs from a failed installation
+
+If you gave an SSH key to your installer, you can gather data about your failed
+installation.
+
+[NOTE]
```

Comment:
@sferich888, based on the discussion from https://github.com/openshift/openshift-docs/pull/14811, do you think that this note is appropriate?